sanic/app.py

```diff
@@ -548,12 +548,13 @@ def run(self, host="127.0.0.1", port=8000, debug=False, before_start=None,
                 warnings.simplefilter('default')
             warnings.warn("stop_event will be removed from future versions.",
                           DeprecationWarning)
+        signal = Signal()
         server_settings = self._helper(
             host=host, port=port, debug=debug, before_start=before_start,
             after_start=after_start, before_stop=before_stop,
             after_stop=after_stop, ssl=ssl, sock=sock, workers=workers,
             loop=loop, protocol=protocol, backlog=backlog,
-            register_sys_signals=register_sys_signals)
+            register_sys_signals=register_sys_signals, signal=signal)
```

Comment:
what is the reasoning for adding this as an argument in both cases, why not just do `signal=Signal()` within the helper and do it once?